index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://kit.fontawesome.com/045dcd1183.js" crossorigin="anonymous"></script>
+  <title>Jeff Kayombo- Portfolio</title>
+</head>
+<body>
+  <nav>
+    <div class="logo">
+      <h2><a href="#">Welcome</a></h2>
+    </div>
+    <div class="nav-menu">
+      <i class="fas fa-bars"></i>
+    </div>
+  </nav>
```

Comment:
- [ ] <b>REQUIRED:</b> As it is stated in project specifications, make sure to follow semantic HTML rule by using `header` tag instead of `nav` tag as the container of the toolbar.